practices/backlog_cleanup.md

```diff
@@ -0,0 +1,60 @@
+---
+slug: backlog-cleaning
+stages:
+  - development
+short_description: Cleaning the product backlog is essential to keep track of which key product features should be developed next. 
+tags:
+  - backlog cleaning
+  - agile
+  - scrum
+keywords:
+  - backlog
+  - cleaning
+  - grooming
+  - agile
+  - scrum
+author_github_username: cadamini
+---
+
+# Backlog Cleanup
+
+**TL;DR**
+
+A well-managed backlog results in a better overview of your important open issues and features, faster planning, and a cleaner scope. Periodically remove obsolete items from the backlog that have been created throughout the project lifecycle.
+
+## What Is Backlog Cleanup  
+
+Backlog Cleanup describes the process of managing issues or ideas for new features in your product backlog. After some strategy meetings in the last months/quarters, some of the items in your backlog may be out of scope, since the product strategy has shifted in a different direction.
+
+![Product backlog](/files/backlog_cleanup.webp)
+[Name](https://img.freepik.com/free-photo/close-up-sticky-note-with-product-backlog_23-2148513829.jpg)
+
+## Why You Might Want Backlog Cleanup
+
+Both the prioritiation and the size of a product backlog is key. Your team will most probably work on the top items in the list. 
+When backlogs grow, it will be harder to see the relationship between items. Backlog cleanup will help you to keep track.
+
+## Problems Backlog Cleanup Helps to Solve
+
+- Getting a better overview of important tasks
+- Not loosing time due to a long and confusing backlog
+
+## How to Implement Backlog Cleanup
+
+- **Go through the product backlog on a regular basis:** For example, once a week or month: This process is called Grooming. The frequency depends on your the number of new cards and how often strategy changes happen.
+- **Find a good upper limit:** Find a good number of backlog items that you can deal with. You can store your ideas separately and review them after your next strategy meeting. 
```

Comment:
```suggestion
- **Define an upper limit for the number of items:** You can store your ideas separately and review them after your next strategy meeting. 
```

Not sure about this point. Now duplicated in pain points and rewritten here. What do you think? Worth the repetition?

Reply:
I don't think repetition is a problem. :) I'll leave it up to you.